inspec/aws/README.md

```diff
@@ -0,0 +1,38 @@
+Gather inventory of resources from AWS environment
+
+You need python 2 or 3
+pip install aws-list-all
+
+List all resources in an AWS account, all regions, all services(*). Writes JSON files for further processing.
+
+mkvirtualenv -p $(which python3) aws
```

Comment:
`$(command -v python3)` I think is better for checking the path of an executable command, but this a README, this is just a suggestion.